Lombiq.Tests.UI/MonkeyTesting/IMonkeyTestingUrlFilter.cs

```diff
@@ -0,0 +1,20 @@
+using Lombiq.Tests.UI.Services;
+
+namespace Lombiq.Tests.UI.MonkeyTesting
+{
+    /// <summary>
+    /// An URL filter that is used in monkey testing.
+    /// </summary>
+    public interface IMonkeyTestingUrlFilter
+    {
+        /// <summary>
+        /// Determines whether this filter allows the specified URL to be tested.
+        /// </summary>
+        /// <param name="url">The URL.</param>
+        /// <param name="context">The context.</param>
+        /// <returns>
+        ///   <see langword="true"/> if URL passes the filter; otherwise, <see langword="false"/>.
+        /// </returns>
```

Comment:
No need for indentations. Also can be a single line:
```suggestion
        /// <returns><see langword="true"/> if URL passes the filter; otherwise, <see langword="false"/>.</returns>
```